.azure-pipelines/generate-v1.0-sdk.yml

```diff
@@ -132,6 +136,26 @@ steps:
   inputs:
     diagnosticsEnabled: True
 
+- task: PowerShell@2 # Setup environment variables and make them available to all tasks. See [1] for more info.
+  displayName: 'Get DLL path'
+  inputs:
+    targetType: inline
+    script: |
+      $allGeneratedDlls = Get-ChildItem $dir -Include Microsoft.Graph.dll -Recurse
+      $dotNetFrameworkDll = $allGeneratedDlls | Where-Object { !$_.FullName.Contains("obj") -and !$_.FullName.Contains("netstandard") }
+      $dllPath = $dotNetFrameworkDll.FullName
+      Write-Host "Path to Microsoft.Graph.dll: $dllPath"
+      Write-Host "##vso[task.setvariable variable=dllPath]$dllPath"
+
+- task: PowerShell@2
+  displayName: 'Generate type summary'
+  inputs:
+    targetType: filePath
+    filePath: '$(Build.SourcesDirectory)/msgraph-sdk-dotnet/scripts/generateTypeSummary.ps1'
+    arguments: '-dllPath $(dllPath) -outputPath $(typeSummaryFilePath)'
```

Comment:
We should make the summary file a published artifact so that we can review it from the pipeline run.

Reply:
I am publishing it in the repo